src/main/java/org/codehaus/groovy/control/CompilerConfiguration.java

```diff
@@ -469,6 +472,8 @@ public CompilerConfiguration() {
         handleOptimizationOption(GROOVYDOC, getSystemPropertySafe("groovy.attach.groovydoc"));
         handleOptimizationOption(RUNTIME_GROOVYDOC, getSystemPropertySafe("groovy.attach.runtime.groovydoc"));
         handleOptimizationOption(PARALLEL_PARSE, getSystemPropertySafe("groovy.parallel.parse", "true"));
+        handleOptimizationOption(ANALYZE_DEAD_CODE, getSystemPropertySafe("groovy.analyze.deadcode", "true"));
+
```

Comment:
`"groovy.branch.analysis"` or `"groovy.dead.code.analysis"`?  Are you sure we want to default to enabled right out the gate?

Spurious extra line added.